js/bitmart.js

```diff
@@ -2230,10 +2240,78 @@ module.exports = class bitmart extends Exchange {
         return this.parseTransactions (records, currency, since, limit);
     }
 
+    async fetchDeposit (id, code = undefined, params = {}) {
+        await this.loadMarkets ();
```

Comment:
as `fetchDeposit` and `fetchWithdraw` are so similar. I wasn't sure if it was better to leave like this or follow the DRY principle and create a `fetchTransaction` function

Reply:
We should rename `fetchWithdraw` > `fetchWithdrawal`, by analogy with `fetchDeposits` and `fetchWithdrawals` (not `fetchWithdraws`).

Reply:
Sorry I wrote it wrong in the comment, but It's `fetchWithdrawal` in the code.